qwoyn/assetlist.json

```diff
@@ -17,7 +17,27 @@
         "base": "uqwoyn",
         "name": "Qwoyn Blockchain",
         "display": "qwoyn",
-        "symbol": "QWOYN",
+        "symbol": "QNQ",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/qwoyn/images/qwoyn.png"
+        }
+      },
+      {
+        "description": "COHO is the in-game token for the Qwoyn Network game, Cosmic Horizon",
+        "denom_units": [
+          {
+            "denom": "ucoho",
+            "exponent": 0
+          },
+          {
+            "denom": "coho",
+            "exponent": 6
+          }
+        ],
+        "base": "ucoho",
+        "name": "Cosmic Horizon",
+        "display": "coho",
```

Comment:
Does QNC has a different token logo? Or it's actually meant to be the same as QNQ logo?

Reply:
It's intended to remain the same as COHO, not intended to leave the Qwoyn Network, and thus will not be displayed anywhere. If it's a better idea, we could consider removing the logo altogether.